engine/table/src/main/java/io/deephaven/engine/table/impl/TimeTable.java

```diff
@@ -114,4 +134,309 @@ protected void destroy() {
         super.destroy();
         UpdateGraphProcessor.DEFAULT.removeSource(this);
     }
+
+    private static class SyntheticDateTimeSource implements
+            ColumnSource<DateTime>,
+            DefaultChunkSource.WithPrev<Values>,
+            MutableColumnSourceGetDefaults.LongBacked<DateTime>,
+            FillUnordered,
+            InMemoryColumnSource {
```

Comment:
This is not an appropriate marker interface to use. This is more like a view column source than an in-memory column source.